src/test/java/core/basesyntax/service/impl/TransactionParserImplTest.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+class TransactionParserImplTest {
+    private final TransactionParserImpl transactionParserImpl = new TransactionParserImpl();
```

Comment:
initialize this field in a method with annotation BeforeAll

```suggestion
    private static TransactionParserImpl transactionParserImpl = new TransactionParserImpl();
```

Reply:
Done. Thank you!